use-cases/demos/Makefile

```diff
@@ -43,5 +51,14 @@ run_object_detection: gen_object_detection
 down_object_detection:
 	@PIPELINE_COUNT=$(PIPELINE_COUNT) RETAIL_USE_CASE_ROOT="$(RETAIL_USE_CASE_ROOT)" RESULTS_DIR="$(RESULTS_DIR)" docker compose -f add_demo_object_detection.yml down
 
-compose_down_all: down_classification down_instance_segmentation down_object_detection
+gen_pytorch_object_detection:
+	@PIPELINE_COUNT=$(PIPELINE_COUNT) RETAIL_USE_CASE_ROOT="$(RETAIL_USE_CASE_ROOT)" RESULTS_DIR="$(RESULTS_DIR)" docker compose -f add_demo_pytorch_object_detection.yml convert > $(DOCKER_COMPOSE)
+
+run_pytorch_object_detection: gen_pytorch_object_detection render
+	@PIPELINE_COUNT=$(PIPELINE_COUNT) RETAIL_USE_CASE_ROOT="$(RETAIL_USE_CASE_ROOT)" RESULTS_DIR="$(RESULTS_DIR)" docker compose -f $(DOCKER_COMPOSE) up -d
```

Comment:
can we take INPUTSRC env so that the docker-compose file is not hard-coded with the media file?

Reply:
umm..where do you suggest adding it ? We don't need rtsp streaming for this simple flow. Also media gets added to docker-compose from add_dependent_services.yml for other flows. I am following the same flow except for using rtsp streaming.

Reply:
take INPUTSRC=$(INPUTSRC) from the env when the compose is generated i think, and in the docker-compose file, you will have to make the volume from the hostpath to the container-path

Reply:
nvm, i just saw you already have that volume mapped in compose file, so just a matter of taking it from env in the Makefile